exporter/datadogexporter/metadata/metadata.go

```diff
@@ -58,6 +61,14 @@ type HostMetadata struct {
 type HostTags struct {
 	// OTel are host tags set in the configuration
 	OTel []string `json:"otel,omitempty"`
+
+	// System are host tags from EC2.
+	// For compatibility with the Datadog Agent we use the
+	// system host tags to send EC2 host tags
```

Comment:
This can be checked here https://github.com/DataDog/datadog-agent/blob/491c309e374ed5c7f1b385b347d5f9b5ac72c2b6/pkg/metadata/host/host_tags.go#L97-L139

I am a bit worried that this will cause trouble if an Agent and a Collector are running at the same time